### DISCUSSION TITLE
Ask Kubernetes Compatibility Matrix for csi-wekafs

### DISCUSSION THREAD

**Discussion by jslouisyou** [weka/csi-wekafs#254] [Q&A] [answered]
Hi, WEKA team.

Is there any Kubermetes Compatibility Matrix for csi-wekafs plugin?
I mean, even if I use very old version of Kubernetes but still can use latest version of csi-wekafs plugin?

Please let me know if there are any limitations or prerequisites.

Thanks!

**Comment by sergeyberezansky** [comment, marked as answer]
Hello,
The WEKA CSI plugin relies on the CSI standard specifications and is bound by its lifecycle.
Current version is up to date with CSI Spec v1.10.
